docs/guides/cloud/branch-review.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Branch Review
+sidebar_position: 35
+sidebar_class_name: new_label
+---
+
+:::info
+
+&#8239;<Icon name="exclamation-triangle" /> Branch Review is currently in beta.
+See [how to enable](/guides/cloud/branch-review#Public-Beta-Access) this feature below.
+
+:::
+
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What Branch Review is and why you should use it
+- What test insights are available to review between branches
+- How Branch Review works and what might impact available data
+
+:::
+
+Cypress Cloud users can review which tests in their current branch are failed, flaky, pending, added, or modified compared to their base branch.
+
+Previously, pinpointing changes in your test suite's results required a manual side-by-side comparison between your newly-introduced branch's test runs and your base branch's test runs. This sub-optimal workflow often fails to answer the fundamental questions, _what changed_ and _why?_ You might be left wondering if the same tests are flaky between branches, when new failures were introduced, or if you added sufficient test coverage to your new branch.
+
+Cypress Branch Review is designed to elevate your pull request (PR) review workflow. It allows you to quickly identify the impact a pull request might have on your test suite in a single view. Compare which tests are failing, flaky, pending, added, or modified between the **<u>source</u>** and **<u>base</u>** branches and prevent the merging of low-quality code.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review.jpg"
+  alt="Cypress Branch Review"
+/>
+
+:::info
+
+Cypress Branch Review is currently only available for [GitHub integrated projects](/guides/cloud/integrations/source-control/github). We are working on adding support for GitLab and Bitbucket soon.
+
+:::
+
+## Public Beta Access
+
+:::tip
+
+To access the Branch Review public beta, an admin can simply enable it under "Cypress Labs"<br />in your Organization Settings in Cypress Cloud.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/labs-beta.png"
+  alt="Enable Branch Review Beta"
+  width="90%"
+/>
+
+:::
+
+## Getting Started
+
+Branch Review works by leveraging the Cypress Cloud GitHub integration to query the GitHub API for branches with pull requests.
+
+Your project will first need to be connected to a GitHub repository. If you haven't already, follow the steps in [Install the Cypress GitHub app](/guides/cloud/integrations/source-control/github#Install-the-Cypress-GitHub-app) to connect your project to GitHub.
+
+### Accessing Branch Review
+
+To access Branch Review, navigate to the **Branches** tab in the left sidebar and select the branch you want to review. In order to see a comparison, the branch you select must have an **open** pull request.
```

Comment:
all PRs have comparisons if we can find runs for them, not just open ones – for example, if you just merged a PR, you can still go back and see the comparison